CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ Features:
 - [#1340](https://github.com/rails-api/active_model_serializers/pull/1340) Add support for resource-level meta. (@beauby)
 
 Fixes:
+- [#1622](https://github.com/rails-api/active_model_serializers/pull/1622) Fix fragment caching inherited serializers to use distinct per-serializer caches. (@lserman)
```

Comment:
I'm not too happy with this sentence though. Hard to understand what was the actual change. I'd prefer something closer to the PR title e.g. `Fragment cache now fragments based on the serializer name instead of the model name`.

@bf4 do you agree or am I being too picky here? :smile: